src/service/CommandUtils.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { flags } from '@oclif/command';
+import { IOptionFlag } from '@oclif/command/lib/flags';
+import { prompt } from 'inquirer';
+import { LogType } from '../logger';
+import Logger from '../logger/Logger';
+import LoggerFactory from '../logger/LoggerFactory';
+import { BootstrapUtils } from './BootstrapUtils';
+const logger: Logger = LoggerFactory.getLogger(LogType.System);
+
+export class CommandUtils {
+    public static passwordPromptDefaultMessage = `Enter the password used to encrypt and decrypted custom presets, addresses.yml, and preset.yml files. When providing a password, private keys would be encrypted. Keep this password in a secure place!`;
+    public static helpFlag = flags.help({ char: 'h', description: 'It shows the help of this command.' });
+
+    public static targetFlag = flags.string({
+        char: 't',
+        description: 'The target folder where the symbol-bootstrap network is generated',
+        default: BootstrapUtils.defaultTargetFolder,
+    });
+
+    public static passwordFlag = CommandUtils.getPasswordFlag(
+        `A password used to encrypt and decrypted custom presets, addresses.yml, and preset.yml files. When providing a password, private keys would be encrypted. Keep this password in a secure place!`,
```

Comment:
```suggestion
        `A password used to encrypt and decrypt private keys in preset files like addresses.yml and preset.yml. Password can be provided in the command line (--password=XXXX) or requested in a prompt (--password).`,
```

Reply:
What about. 

> A password used to encrypt and decrypt private keys in preset files like addresses.yml and preset.yml. Bootstrap prompts for a password by default, can be provided in the command line (--password=XXXX) or disabled in the command line (--noPassword).

Prompt is used by default.